Library/Util.ts

```diff
@@ -140,10 +140,11 @@ class Util {
                 var property = obj[field];
                 var propertyType = typeof property;
                 if (propertyType !== "string") {
-                    if (property && typeof property.toString === "function") {
+                    if (property != null && typeof property.toString === "function") {
```

Comment:
[TypeScript wiki](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined) seems to want us use undefined.
Was it changed recently or we only care about null here?